### PR TITLE
Updating openfpga related synthesis flow

### PR DESCRIPTION
**Pull request by lnsharma**

### CODE REVIEW

File: techlibs/quicklogic/synth_quicklogic.cc [hunk 1]
```diff
@@ -244,7 +248,9 @@ struct SynthQuickLogicPass : public ScriptPass {
                 techMapArgs += "_ffs_map.v";
             }
 
-            run("techmap " + techMapArgs);
+            if(!openfpga) {
+                run("techmap " + techMapArgs);
+            }
```
Comment (kgugala) [inline techlibs/quicklogic/synth_quicklogic.cc:253]: don't we need any techmap for openfpga?

Comment (lnsharma) [reply techlibs/quicklogic/synth_quicklogic.cc:253]: In openfpga Caraval architecture, by default latches are inferred  and my objective is to generate blif that can directly be used in openfpga flow. So, this doesn't require mapping FF and the above marked code is for tech-mapping ff.

Comment (kgugala) [reply techlibs/quicklogic/synth_quicklogic.cc:253]: ok

**Review by kgugala — APPROVED** [review]
LGTM

**Pull request merged**